README.md

```diff
@@ -15,6 +15,8 @@ Make sure $GOPATH/bin is in your $PATH, then:
 
 ## Building
 
+Requires [glide](https://github.com/Masterminds/glide) for dependency management
+
     go get -u github.com/dcu/mongodb_exporter
```

Comment:
`go get -d -u github.com/dcu/mongodb_exporter`? Otherwise it would try to build and still fail I think.